enforcer-parent/enforcer/pom.xml

```diff
@@ -404,6 +404,11 @@
             <groupId>org.wso2.carbon.apimgt</groupId>
             <artifactId>org.wso2.carbon.apimgt.common.jms</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.am.analytics.publisher</groupId>
+            <artifactId>org.wso2.am.analytics.publisher.client</artifactId>
+            <version>1.0.8</version>
```

Comment:
Add this dependency to the parent pom and refer from there.

Reply:
test-ingratiation pom was using 1.0.4 version and after updating the parent pom tests started fail. Therefore I updated the version in enforcer pom to 1.0.8

Reply:
Let's check the reasons for that.. We can't just add it here.